src/main/webapp/app/exercises/quiz/manage/short-answer-question/short-answer-question-edit.component.ts

```diff
@@ -450,7 +463,11 @@ export class ShortAnswerQuestionEditComponent implements OnInit, OnChanges, Afte
         this.textParts = this.shortAnswerQuestionUtil.divideQuestionTextIntoTextParts(questionText);
         const textOfSelectedRow = this.textParts[selectedTextRowColumn[0]][selectedTextRowColumn[1]];
         this.textParts[selectedTextRowColumn[0]][selectedTextRowColumn[1]] =
-            textOfSelectedRow.substring(0, startOfRange) + '[-spot ' + this.numberOfSpot + ']' + textOfSelectedRow.substring(endOfRange);
+            textOfSelectedRow.substring(0, startOfRange) +
+            `[-spot${type === SpotType.NUMBER ? '-number' : ''} ` +
+            this.numberOfSpot +
+            ']' +
```

Comment:
Why not use a format string for the whole line?

Reply:
I changed only the part that I needed. But I've just updated the whole line to use string interpolation instead.